### PR TITLE
LPS-152781 [CP 2.0] - Add CP2MultipleIndividualKeys tests

### PR DESCRIPTION
**Pull request by MathausC**
https://issues.liferay.com/browse/LPS-152781

### CODE REVIEW

**Comment by MathausC** [comment]
ci:test:stable

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2ActivationKeys.macro [hunk 1]
```diff
@@ -1,5 +1,23 @@
 definition {
 
+	macro assertDropDownItemIsDesabled {
```
Comment (barrosdennys) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2ActivationKeys.macro:3]: @MathausC please check the english here. Please change "desabled" to "disabled"

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2ActivationKeys.macro [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+		var key_dropDownItem = "${dropdownItem}";
+
+		AssertElementPresent(locator1 = "CP2ActivationKeys#DESABLED_DROPDOWN_ITEM_BUTTON");
```
Comment (barrosdennys) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2ActivationKeys.macro:6]: @MathausC please check the english here. Please change "desabled" to "disabled"

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/paths/CustomerPortalSite.path [hunk 3]
```diff
@@ -295,6 +295,21 @@
...
 </tr>
+<tr>
+	<td>LIFERAY_SITE</td>
+	<td>//*[contains(@content, 'https://www.liferay.com')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>SEARCH_BAR</td>
+	<td>xpath=(//*[contains(@class, 'search')])[1]</td>
+	<td></td>
```
Comment (barrosdennys) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/paths/CustomerPortalSite.path:306]: Is there a way to retrieve this element without using index?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2ActivationKeys.macro [hunk 4]
```diff
@@ -1,5 +1,23 @@
...
+	}
+
+	macro assertDropDownItemIsEnabled {
```
Comment (barrosdennys) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2ActivationKeys.macro:9]: Mathaus, please change all occurrences of "DropDown" to "Dropdown" or "dropdown" in order to keep consistency.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/paths/CP2ActivationKeys.path [hunk 5]
```diff
@@ -25,6 +25,36 @@
 	<td>//div[@class='table-responsive']//td[contains(.,'${key_text}')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>DROPDOWN_ITEM_BUTTON</td>
+	<td>//*[contains(@class, 'dropdown-item')]//*[text()='${key_dropDownItem}']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>DESABLED_DROPDOWN_ITEM_BUTTON</td>
```
Comment (barrosdennys) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/paths/CP2ActivationKeys.path:34]: Please change "DESABLED" to "DISABLED"

**Comment by MathausC** [comment]
ci:test:sf

**Comment by MathausC** [comment]
ci:test:stable

**Comment by barrosdennys** [comment]
LGTM

**Comment by barrosdennys** [comment]
ci:forward

**Pull request closed without merging**